packet/loadbalancers.go

```diff
@@ -137,7 +137,7 @@ func (l *loadBalancers) reconcileNodes(nodes []*v1.Node, remove bool) error {
 				klog.Errorf("could not add metallb node peer address for node %s: %v", node.Name, err)
 				continue
 			}
-			config = addNodePeer(config, node.Name, peers[0], l.localASN, l.peerASN)
+			config = addNodePeer(config, node.Name, peers, l.localASN, l.peerASN)
```

Comment:
It looks like we are assuming that all peers share the same ASN, is that intentional?

Reply:
Yes. For now, that is what I have seen in every single use case; I never really have seen someone try to do something different. If it comes up (and Packet actually supports it), we can create that option then.